website_img_bg_style/views/snippets.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015 Grupo ESOC Ingeniería de Servicios, S.L.U. - Jairo Llopis
+     Copyright 2016 Tecnativa - Vicent Cubells
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+
+<template id="snippet_options"
+          inherit_id="website.snippet_options"
+          name="Container width type chooser">
+    <xpath expr=".">
+        <div data-js='background-style'
+             data-selector="section, :not(.o_gallery > .container) > .carousel, .parallax">
```

Comment:
Ain't it better here to simply put `.oe_img_bg`? Blocks without background image do not need any of these customizations...

OTOH, another possibility is to put these new options under *Background > Style*, to remove apparent duplicities as you can see in the image:

![captura de pantalla de 2016-12-14 09-16-43](https://cloud.githubusercontent.com/assets/973709/21174776/674ee92c-c1df-11e6-9320-203fa41c8089.png)